### DISCUSSION TITLE
rows multiply automatically when collapse row

### DISCUSSION THREAD

**Discussion by adilvalizada** [TanStack/table#5464] [Q&A]
I have a problem. I am using React-table with rows and subrows. Whenever I expand my rows, the table automatically duplicates the subrows. What could be the reason for this?

https://github.com/TanStack/table/assets/71330310/e619114f-203d-467d-86fc-13a11344ee4b

<img width="551" alt="V1iFg" src="https://github.com/TanStack/table/assets/71330310/5af3542e-877b-4ee5-b329-16c7e0643603">

This is my Table configuration.

**Comment by goddev1026** [comment]
Hello @adilvelizade0 ,

Could you share a full codebase so that I can check?

Thanks

**Comment by amgoun** [comment]
Hi, would you provide full code or at least code of your table component
